CMakeLists.txt

```diff
@@ -379,7 +379,13 @@ endif()
 
 configure_file(libiio.iss.cmakein ${CMAKE_CURRENT_BINARY_DIR}/libiio.iss @ONLY)
 
+include(cmake/JoinPaths.cmake)
+
+join_paths(libdir_for_pc_file "\${prefix}" "${CMAKE_INSTALL_LIBDIR}")
+join_paths(includedir_for_pc_file "\${prefix}" "${CMAKE_INSTALL_INCLUDEDIR}")
```

Comment:
continuing from comment https://github.com/analogdevicesinc/libiio/pull/563/commits/48e1606c30b5ae567aee55ec7bb6765d0e5b79ac#r449998108
this is where we could do:
```
if(NOT IS_ABSOLUTE "${CMAKE_INSTALL_LIBDIR}")
CMAKE_INSTALL_LIBDIR=${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}
endif
```